views.py

```diff
@@ -75,25 +75,28 @@ def mentors():
             return 'Mentor exists (email)', 409
 
 
-@app.route('/mentors/<mentor_id>', methods=['GET', 'PUT'])
-def get_mentor_by_id(mentor_id):
+@app.route('/mentors/<user_uid>', methods=['GET', 'PUT'])
+def get_mentor_by_uid(user_uid):
```

Comment:
Is that the Firebase uid? If so, we shouldn't use it in the http calls, and use mentor id generated in our DB.